### ISSUE TITLE
Add query of next call permitted state in web app

### ISSUE THREAD

**Issue by whitedavidp** [thess/callattendant#52]
Hi Ted. I see you are currently fiddling with this. While doing so, might I suggest adding the ability to check the next call flag state via the web app? Something like this in webapp.py:

```
@app.route('/callers/querynextcall')
def Callers_query_next_call():
    nextcall = NextCall(app.config['MASTER_CONFIG'])
    if nextcall.is_next_call_permitted():
        return '1Next call will be permitted.'
    else:
        return '0Next call will be handled normally.'
```
It mimics the existing /callers/permitnextcall - just without the actual toggling of the flag's state.

I use this small mod from my Android CA UI. Thanks.

**Comment by thess** [comment]
Yes, I have been _fiddling_ with the `/callers/permitnextcall` request. I added an optional argument `?toggle` to be used to change the state of this indicator. Doing a GET without the argument, will just query the status as you desire. I didn't see a need to add another API call for just this function so I extended the existing request dispatch.

See: https://github.com/thess/callattendant/blob/fmsentry/callattendant/userinterface/webapp.py#L851
